odoo/addons/base/models/res_currency.py

```diff
@@ -52,6 +52,23 @@ class Currency(models.Model):
         ('rounding_gt_zero', 'CHECK (rounding>0)', 'The rounding factor must be greater than 0!')
     ]
 
+    def write(self, vals):
+        res = super().write(vals)
+        if 'active' not in vals:
+            return res
+        # Automatically activate group_multi_currency if there is more than 1 active currency, deactivate it otherwise
+        group_user = self.env.ref('base.group_user', raise_if_not_found=False)
+        group_mc = self.env.ref('base.group_multi_currency', raise_if_not_found=False)
+        group_pp = self.env.ref('product.group_product_pricelist', raise_if_not_found=False)
```

Comment:
Nope, you shouldn't have code referring to a `product` module data in `base`.
You should cover this case in the product module on currency update/creation/deletion (do not cover the module installation case, it's not worth it IMHO).

Reply:
removed this into `_toggle_group_multi_currency` in `res.config.settings`